From 656576670e1c4b94f23b921ed70749d834e32b35 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Sat, 19 Sep 2020 21:22:28 -0700
Subject: [PATCH 11/15] Screenshot: Reduce screenshot dismiss delay to 3
 seconds

3 seconds is more than enough for most cases. Keeping the overlay around
for too long forces the user to dismiss it manually, which is a
regression compared to the old screenshot notification UI.

Change-Id: I5644a055d1285d390d77430670fd6f43862c9608
Signed-off-by: Adithya R <gh0strider.2k18.reborn@gmail.com>
---
 .../com/android/systemui/screenshot/ScreenshotController.java   | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/screenshot/ScreenshotController.java b/packages/SystemUI/src/com/android/systemui/screenshot/ScreenshotController.java
index c213f192291a..7ef88d7dcb7e 100644
--- a/packages/SystemUI/src/com/android/systemui/screenshot/ScreenshotController.java
+++ b/packages/SystemUI/src/com/android/systemui/screenshot/ScreenshotController.java
@@ -242,7 +242,7 @@ public class ScreenshotController {
     // From WizardManagerHelper.java
     private static final String SETTINGS_SECURE_USER_SETUP_COMPLETE = "user_setup_complete";
 
-    private static final int SCREENSHOT_CORNER_DEFAULT_TIMEOUT_MILLIS = 6000;
+    private static final int SCREENSHOT_CORNER_DEFAULT_TIMEOUT_MILLIS = 3000;
 
     private final WindowContext mContext;
     private final ScreenshotNotificationsController mNotificationsController;
-- 
2.34.1

